src/Components/ServiceScene/JSONPanel/GetItemString.tsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+
+import { rootNodeItemString } from '../../../services/JSONViz';
+import { hasProp } from '../../../services/narrowing';
+import { JsonDataFrameTimeName, JsonVizRootName } from '../LogsJsonScene';
+import { KeyPath } from '@gtk-grafana/react-json-tree/dist/types';
+
+interface GetItemStringProps {
+  data: unknown;
+  itemString: string;
+  itemType: React.ReactNode;
+  keyPath: KeyPath;
+  nodeType: string;
+}
+export default function GetItemString({ data, itemString, itemType, keyPath }: GetItemStringProps) {
```

Comment:
Minor, but what's the purpose of the "Get" prefix? If it's just to render an ItemString, `<ItemString />` should be enough.